guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -138,3 +138,15 @@ In such a case, change directory, for example to the *_root_* user's home direct
 ----
 # {package-install-project} tfm-rubygem-smart_proxy_discovery.noarch
 ----
+. Remove all Pulp2 content on each smart proxy.
```

Comment:
Will Sat customers have to do this step in time or will it be upstream-only?

Reply:
I beleive it's for both upstream and down, but I will check.

Reply:
@melcorr For now there is no difference between upstream and down.